integration_test.go

```diff
@@ -94,7 +115,7 @@ func TestPipeline(t *testing.T) {
 
 	//Wait for fluent-bit connection to generate data; add delays before ending fluent-bit process
 	time.Sleep(2 * time.Second)
-	if err := generateData(numRows, time.Second, false); err != nil {
```

Comment:
Is `generateData` being called from anywhere else? Maybe it can be deleted?

Reply:
We call generateData in three tests and then call generateAllDataTypes in the fourth test. This actually sends the data to the sink (log file).